src/containers/contact/Contact.js

```diff
@@ -1,29 +1,31 @@
-import React from "react";
+import React,{useContext} from "react";
 import "./Contact.css";
 import SocialMedia from "../../components/socialMedia/SocialMedia";
-import {contactInfo} from "../../portfolio";
+import { contactInfo } from "../../portfolio";
 import { Fade } from "react-reveal";
+import StyleContext from "../../contexts/StyleContext";
 
 export default function Contact() {
+    const { isDark } = useContext(StyleContext);
     return (
         <Fade bottom duration={1000} distance="20px">
         <div className="main contact-margin-top" id="contact">
             <div className="contact-div-main">
                 <div className="contact-header">
                     <h1 className="heading contact-title">{contactInfo.title}</h1>
-                    <p className="subTitle contact-subtitle">{contactInfo.subtitle}</p>
+                    <p className={isDark ? "dark-mode contact-subtitle":"subTitle contact-subtitle"}>{contactInfo.subtitle}</p>
 
-                    <div className="contact-text-div">
+                    <div className={isDark ? "dark-mode contact-text-div" : "contact-text-div"}>
                         <a className="contact-detail" href={"tel:" + contactInfo.number}>{contactInfo.number}</a>
                         <br/><br/>
                         <a className="contact-detail-email"
                            href={"mailto:" + contactInfo.email_address}>{contactInfo.email_address}</a>
                         <br/><br/>
-                        <SocialMedia/>
+                        <SocialMedia />
                     </div>
                 </div>
                 <div className="contact-image-div">
-                    <img alt="Saad Working" src={require("../../assets/images/contactMail.webp")}></img>
+                    <img alt="Saad Working" src={require(`../../assets/images/${isDark ? 'contactMailDark.svg' : 'contactMail.webp'}`)}></img>
```

Comment:
**Question:** Why not we use SVG for both dark and light mode?

Reply:
We have webp for light mode so we can you that and for dark mode, if we are using SVG because we don't have webp image